lib/utils.js

```diff
@@ -13,9 +13,9 @@ var SW_SHOWNORMAL = 0x1;
 
 module.exports = {
     elevate: function (filepath, parameters, callback) {
-        if(parameters === null || parameters === '') {
-            debug('Missing parameters filepath');
-            callback('Missing parameters filepath');
+        if(filepath === null || filepath === '') {
```

Comment:
check if `undefined `as well.

You can simply do if(`filepath`) and you'll get all of those checks in one.